app/models/manageiq/providers/workflows/automation_manager/authentication.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::Workflows::AutomationManager::Authentication < ManageIQ::Providers::AutomationManager::Authentication
+  validates :name, :presence => true, :uniqueness_when_changed => {:scope => [:tenant_id]},
+            :format => {:with => /\A[\w\-]+\z/i, :message => N_("may contain only alphanumeric and _ - characters")}
```

Comment:
NOTE leave name as a user friendly description and use `#ems_ref` as the "unique" value for the purposes of querying from the workflow Credentials reference